.idea/vcs.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/../Klubik-Multy-Core" vcs="Git" />
```

Comment:
Are you sure that we are going to use that directory name for checked out Multy-Core repo?